docs/src/main/tut/docs.md

```diff
@@ -125,6 +124,33 @@ object ProgramTask {
 
 Note that you'll need a dependency to the `github4s-scalaz` pacakge to leverage `scalaz.Task`.
 
+### Using `cats.effect.IO`
+
+On the JVM:
+```scala
```

Comment:
I can't seem to get this example to compile with tut despite adding cats-effect to the doc deps, any ideas?